README.md

```diff
@@ -67,6 +75,7 @@ POSTGRES_PORT=5432
 POSTGRES_DB=battlesnakeio_play
 POSTGRES_USER=battlesnakeio
 POSTGRES_PASSWORD=battlesnakeio
+ENV=local
```

Comment:
so you don't want this in your .env file, because that won't be picked up unless you specify it on the command line, so this is redundant